octobot_trading/personal_data/orders/order.py

```diff
@@ -633,7 +633,7 @@ def to_dict(self):
             enums.ExchangeConstantsOrderColumns.AMOUNT.value: self.origin_quantity,
             enums.ExchangeConstantsOrderColumns.COST.value: self.total_cost,
             enums.ExchangeConstantsOrderColumns.QUANTITY_CURRENCY.value: self.quantity_currency,
-            enums.ExchangeConstantsOrderColumns.FILLED.value: self.filled_quantity,
+            enums.ExchangeConstantsOrderColumns.FILLED_AMOUNT.value: self.filled_quantity,
```

Comment:
we will have to make sure everything using this will be updated before we release this